content/applications/finance/accounting/reporting/budget.rst

```diff
@@ -1,102 +1,147 @@
+
+Check an analytic budget
+------------------------
+
+Once the budget is in the :guilabel:`Open` status, two additional columns are available:
+:guilabel:`Committed` and :guilabel:`Achieved`. Their amounts are automatically calculated based on
+the vendor bills/invoices and their related analytic distribution. They evolve when a new journal
+entry linked to the analytic account is created. The :guilabel:`Achieved` amount reflects the
+current result according to the expenses or income for the associated analytic account. In contrast,
+the :guilabel:`Committed` amount displays what has already been purchased or sold, regardless of
+whether it has been billed or invoiced.
 
 .. note::
-   Each budgetary position can have any number of accounts from the chart of accounts, though it
-   must have at least one.
+   For budgets in the :guilabel:`Open` status, when a purchase order created with the associated
+   analytic distribution exceeds the amount budgeted, the related purchase order line turns red.
 
-Use case
-========
+To reveal the :guilabel:`Theoretical` amount or percentage, use the :icon:`oi-settings-adjust`
+(:guilabel:`settings adjust`) icon in the far right of the budget top row. The
+:guilabel:`Theoretical` represents the amount of money that could theoretically have been spent or
+should have been received based on the date.
+To open the reporting view, click :guilabel:`Details` and filter the budget lines and columns of
+the pivot table.
 
-Let’s illustrate this with an example.
+.. image:: budget/budget.png
+   :alt: open budget with committed, achieved and theoretical amounts
 
-We just started a project with *Smith & Co*, and we would like to budget the income and expenses of
-that project. We plan on having a revenue of 1000, and we don’t want to spend more than 700.
+.. note::
+   Deletion of a budget is only allowed in the :guilabel:`Draft` and :guilabel:`Cancelled` stages.
 
-First, we need to define what accounts relate to our project’s expenses. Go to
-:menuselection:`Accounting --> Configuration --> Management: Budgetary positions`, and click
-:guilabel:`New` to add a position. Add the accounts wherein expenses will be booked.
+To view all the different budget lines directly from the :guilabel:`Budgets` view, select the
+:guilabel:`Budget Name` and click :guilabel:`Budget Lines`.
 
-.. image::  budget/smith-and-co-expenses.png
-   :align: center
-   :alt: display the Smith and Co expenses
+.. _accounting/reporting/budgets/analytic-budget-generate:
 
-Let's repeat the steps to create a budgetary position that reflects the revenue.
+Generate periodical budgets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image::  budget/smith-and-co-revenue.png
-   :align: center
-   :alt: display the Smith and Co revenue
+The :guilabel:`Generate` option allows the creation of multiple budget periods. To do so, select the
+budget and click :guilabel:`Generate` to open the :guilabel:`Generate Budget` window. Set the dates,
```

Comment:
@dade-odoo haha, I didn't even know you could use this button without selecting a budget 😄 thanks! 
I see that this feature includes all the accounts from the plan selected when creating these periodic budgets. So it's a problem if you select an existing budget because it doesn't take it into account nor does it split the amounts already in there. So I'll get rid of all this and start it from scratch ;)